### PR TITLE
Add build=quick parameter to pull-kubernetes-e2e-inplace-pod-resize-containerd-main-v2 job

### PR DESCRIPTION
**Pull request by vinaykul**
It looks like with extract=local, we need build=quick. This PR brings the fix.

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -1599,6 +1599,9 @@ presubmits:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221107-33c989e684-master
```
Comment (bobbypage) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1607]: Not 100% sure but do we need to add `--repo=k8s.io/release` similar as here (https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L35C16-L36)?

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 2]
```diff
@@ -1599,30 +1599,38 @@ presubmits:
...
           - --env=KUBE_MASTER_EXTRA_METADATA=user-data=/workspace/github.com/containerd/containerd/test/e2e/master.yaml,containerd-configure-sh=/workspace/github.com/containerd/containerd/cluster/gce/configure.sh,containerd-extra-init-sh=/workspace/github.com/containerd/containerd/test/e2e_node/gci-init.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-main/cgroupv2/env-cgroupv2
           - --env=KUBE_NODE_EXTRA_METADATA=user-data=/workspace/github.com/containerd/containerd/test/e2e/node.yaml,containerd-configure-sh=/workspace/github.com/containerd/containerd/cluster/gce/configure.sh,containerd-extra-init-sh=/workspace/github.com/containerd/containerd/test/e2e_node/gci-init.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-main/cgroupv2/env-cgroupv2
           - --env=KUBELET_TEST_ARGS=--runtime-cgroups=/system.slice/containerd.service --cgroup-driver=systemd
+          - --build=quick
           - --extract=local
           - --gcp-zone=us-west1-b
           - --gcp-node-image=gci
           - --gcp-nodes=1
           - --provider=gce
+          - --runtime-config=api/all=true
```
Comment (bobbypage) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1632]: I think this is specific to the alpha job, I don't think we need this for now.

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 3]
```diff
@@ -1599,30 +1599,38 @@ presubmits:
...
+          - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-inplace-pod-resize-containerd-main-v2
```
Comment (bobbypage) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1633]: I don't think this is needed, this is if you want to save the artifacts from the build, I don't think that's needed here.

Comment (vinaykul) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1633]: sg. i'll drop these two shortly.

**Review by endocrimes — APPROVED** [review]

**Pull request merged**